src/airtable/proposals/proposal_standings.js

```diff
@@ -187,12 +187,15 @@ const getProposalRecord = async (proposal, allProposals) => {
 }
 
 // Returns all Proposal Standings, indexed by Project Name
-const processProposalStandings = async (allProposals) => {
+const processProposalStandings = async (allProposals, extraProposals = []) => {
```

Comment:
Please rename these parameters. It's difficult to understand extraProposals. I imagine they are representing currentRoundProposals and previousRoundsProposals

Reply:
I agree, thanks for pointing that out!